misc-utils/fincore.c

```diff
@@ -489,14 +495,20 @@ int main(int argc, char ** argv)
 	textdomain(PACKAGE);
 	close_stdout_atexit();
 
-	while ((c = getopt_long (argc, argv, "bcno:JrRVh", longopts, NULL)) != -1) {
+	while ((c = getopt_long (argc, argv, "bcCno:JrRVh", longopts, NULL)) != -1) {
 		switch (c) {
 		case 'b':
 			ctl.bytes = 1;
 			break;
 		case 'c':
 			ctl.total = 1;
 			break;
+		case 'C':
+#ifndef HAVE_CACHESTAT
```

Comment:
This `#ifndef` can be removed. We have fallback definitions in place.

Reply:
and just let the syscall fail in case?